form_to_clean_csv.py

```diff
@@ -0,0 +1,23 @@
+import pandas as pd
+import sys
+
+# usage: [form.csv] [hw_number]
+# creates 2 files hw_number_handles.csv & hw_number_teams.csv
```

Comment:
This should probably use `argparse` - you can copy and modify the positional argument code from `canvas-wrangler.py` instead of `sys.argv`.